engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/module/scope/Definition.scala

```diff
@@ -0,0 +1,840 @@
+package org.enso.compiler.core.ir.module.scope
+
+import org.enso.compiler.core.IR
+import org.enso.compiler.core.ir.{DiagnosticStorage, Expression, IRKind, IdentifiedLocation, MetadataStorage}
+import org.enso.compiler.core.ir.module.Scope
+import org.enso.compiler.core.IR.{DefinitionArgument, Identifier, Name, ToStringHelper, indentLevel, mkIndent, randomId}
+
+
+/** A representation of top-level definitions. */
+trait Definition extends Scope {
```

Comment:
Nice reorg. `Scope` is defined one package up and its implementations are in `scope` subpackage.